views/js/jquery.rating.plugin.js

```diff
@@ -81,7 +81,7 @@ jQuery.fn.rating = function(generalOptions) {
       for (var i = minValue; i <= maxValue; ++i) {
         newStar = emptyStar.clone();
         newStar.data('grade', i);
-        newStar.hover(function overStar() {
+        newStar.on('mouseenter mouseleave', function overStar() {
```

Comment:
```suggestion
        newStar.on('mouseenter', function overStar() {
```
You don't need "mouseleave" event here.